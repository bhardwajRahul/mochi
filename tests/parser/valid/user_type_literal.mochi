type Person {
  name: string
  age: int
}
let bob = Person { name: "Bob", age: 30 }
