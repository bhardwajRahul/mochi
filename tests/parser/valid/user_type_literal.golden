(program
  (type Person
    (field name (type string))
    (field age (type int))
  )
  (let bob
    (struct Person
      (field name (string Bob))
      (field age (int 30))
    )
  )
)
